plugins/services/src/js/filters/__tests__/ServiceAttributeIsFilter-test.js

```diff
@@ -73,9 +73,9 @@ describe("ServiceAttributeIsFilter", function() {
     ]);
   });
 
-  it("Should correctly keep services in suspended state", function() {
+  it("Should correctly keep services in stopped state", function() {
```

Comment:
❣️ If we adjust the message anyways let's remove the `Should` `it("keeps services in stopped state"), function() {`

Reply:
yeah I thought about removing the `should` but if I do I need to removed all others `should` too to keep it consistent and since I'm trying to have as least loc changed I decided to keep.

if more people agree in removing all `should` in this file I'm in otherwise I wouldn't remove only for this guy.

@MatApple @ahoskins @bstavroulakis ?

Reply:
Whenever I write new tests, I try to not write `should` because I've been told not to many times.

Reply:
Yes, I agree @ahoskins and I despise using `should` but the point here is SHOULD I remove **ALL** the `should`s?

Reply:
ALL!

Reply:
Well in that case based on both, we would have to also remove the should from this file https://github.com/dcos/dcos-ui/pull/2475/files#diff-06dacf8f6b57206ec2e005a8ebce14a3

Reply:
@weblancaster I value consistency but don't think you should change all test cases. Let's not change the scope of this PR.

Reply:
@orlandohohmeier I agree I will keep the `should` for all for now.

Reply:
@weblancaster what ever you prefer. :)

Reply:
@orlandohohmeier so I decided to open up a separated PR removing the `should` from this file and some others. https://github.com/dcos/dcos-ui/pull/2497

Reply:
@weblancaster awesome! 🙌